src/contractTest/resources/application.properties

```diff
@@ -31,17 +31,17 @@ pagination.page.size=25
 #max idle time in seconds that applies to all definition cache entries
 definition.cache.max-idle.secs=${DEFINITION_CACHE_MAX_IDLE_SEC:259200}
 
-#ttl in seconds for the latest version objects cached
-definition.cache.latest-version-ttl=${DEFINITION_CACHE_LATEST_VERSION_TTL_SEC:30}
+#default ttl in seconds
+default.cache.ttl=${DEFAULT_CACHE_TTL_SEC:30}
 
 #ttl in seconds for the jurisdiction objects cached
 definition.cache.jurisdiction-ttl=${DEFINITION_CACHE_JURISDICTION_TTL_SEC:30}
 
 #max size of cache map in terms of number of entries
-definition.cache.max.size=${DEFINITION_CACHE_MAX_SIZE:5000}
+default.cache.max.size=${DEFAULT_CACHE_MAX_SIZE:5000}
 
-#eviction policy. If set to NONE the cache can grow arbitrarily large and definition.cache.max.size is ignored
-definition.cache.eviction.policy=${DEFINITION_CACHE_EVICTION_POLICY:NONE}
+#eviction policy. If set to NONE the cache can grow arbitrarily large and max size is ignored
+default.cache.eviction.policy=${DEFAULT_CACHE_EVICTION_POLICY:NONE}
```

Comment:
`definition.cache.jurisdiction-ttl` and `definition.cache.max-idle.secs` not updated as per `src/main/resources/application.properties`.    If they need to match may be cut and past the section so order and comments match as well.

Reply:
Sorry: `definition.cache.jurisdiction-ttl` doesn't need to change ...

... but `definition.cache.max-idle.secs` probably should, i.e. to `default.cache.max-idle`.

Reply:
Done. Thank you, Matt.